data/json/items/vehicle/utilities.json

```
@@ -189,9 +189,18 @@
     "id": "fridgetank",
     "name": { "str": "refrigerated tank" },
     "description": "A 60L refrigerated tank for keeping liquids cool.  Provides some insulation from outside weather.",
-    "insulation": 2,
+    "insulation": 10,
     "volume": "60 L",
```

It just needs to be at least 1ml bigger, but figure it has more going on than just being a simple 60L tank.
```suggestion
    "volume": "72 L",
```